server/src/test/java/org/elasticsearch/search/query/PartialHitCountCollectorTests.java

```diff
@@ -101,15 +105,33 @@ public void testCollectedHitCount() throws Exception {
     }
 
     public void testCollectedHitCountEarlyTerminated() throws Exception {
-        Query query = new BooleanQuery.Builder().add(new TermQuery(new Term("string", "a1")), BooleanClause.Occur.SHOULD)
-            .add(new TermQuery(new Term("string", "b3")), BooleanClause.Occur.SHOULD)
-            .add(new TermQuery(new Term("string", "b6")), BooleanClause.Occur.SHOULD)
+        Query query = new BooleanQuery.Builder().add(termQuery(new Term("string", "a1")), BooleanClause.Occur.SHOULD)
+            .add(termQuery(new Term("string", "b3")), BooleanClause.Occur.SHOULD)
+            .add(termQuery(new Term("string", "b6")), BooleanClause.Occur.SHOULD)
```

Comment:
there is another test that uses this same trick. By the way a boolean query is no longer required with your change, you can simplify that and perhaps adjust which fields get populated in the docs that get indexed.